build.gradle

```diff
@@ -20,9 +20,10 @@ plugins {
 
 allprojects {
     repositories {
-        google()
         mavenCentral()
-        mavenLocal()
+        google()
+        // only enable temporarily if needed:
+        / * mavenLocal() */
```

Comment:
Then we can remove it and add it when needed no?

Reply:
This tells you how to add it (so you don't end up hours figuring out what the exact place is), if we remove how to add it, how will people know?